### DISCUSSION TITLE
trying to add the chartingstate.hx from js to regular psych .6.3 what is the type "EditingMusin"

### DISCUSSION THREAD

**Discussion by Memelord111** [JordanSantiagoYT/FNF-JS-Engine#619] [Q&A]
title says it all
source/editors/ChartingState.hx:181: characters 16-28 : Type not found : EditingMusic

**Comment by moxie-coder** [comment]
> title says it all source/editors/ChartingState.hx:181: characters 16-28 : Type not found : EditingMusic

why in fresh hell are you trying to do that, you missed a file for one, and for two, js *is* 0.6.3

**Comment by JordanSantiagoYT** [comment]
> title says it all source/editors/ChartingState.hx:181: characters 16-28 : Type not found : EditingMusic

EditingMusic is just a backend for the background music that plays when you don't do anything for a bit






also js runs on 0.6.3
